SECURITY.md

```diff
@@ -1,5 +1,4 @@
 # Security Policy
 
-## Reporting a Vulnerability
-
-Please report security issues to `security@twistedmatrix.com`
\ No newline at end of file
+treq is covered by the [Twisted project security policy](https://github.com/twisted/twisted/security/policy).
+You can [privately report via GitHub](https://github.com/twisted/treq/security/advisories/new), or via email as described in the policy linked above.
```

Comment:
Very minor comment

Maybe add a newline so that we have 2 paragraphs.

preview here https://github.com/twisted/treq/tree/336-security-docs?tab=security-ov-file